DiagnosticAnalysisLauncher/Models/DiagnosticAnalysis.cs

```diff
@@ -0,0 +1,186 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="DiagnosticAnalysis.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// </copyright>
+// -----------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace DiagnosticAnalysisLauncher
+{
+    public class DiagnosticAnalysis
+    {
+        public Interpretedresult[] interpretedResults { get; set; }
+
+        [JsonIgnore]
+        public Result[] results { get; set; }
+    }
+
+    public class Interpretedresult
```

Comment:
small nitpick in naming convention. Should be InterpretedResult

Reply:
Yes unfortunately. This is an auto-generated class based on the json model generated by Visual Studio Diagnostic tool itself and we cannot change the casing here.